### PR TITLE
VAMC system/facilities selectors, connectors, and tests

### PR DESCRIPTION
**Pull request by eselkin**
## Summary

- Selectors, Connectors, and tests for KISS data from Drupal View in content-build
- Sitewide Facilities

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16019

## Testing done

- Adds testing for preprocess and selectors
- Checked unit tests
- No possibility for e2e test yet until something implements the selectors/connectors

## Screenshots

N/A

## What areas of the site does it impact?

Added new selectors/connectors/preprocessors for KISS data, no existing site functionality changed.

## Acceptance criteria

- [x] Selector is created in vets-website
- [x] Connector is creates in vets-website

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [x] Documentation has been updated (No updated docs)
- [x] Screenshot of the developed feature is added (n/a)
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed (n/a)

### Error Handling

- [x] Browser console contains no warnings or errors.
- [x] Events are being sent to the appropriate logging solution
- [x] Feature/bug has a monitor built into Datadog or Grafana (n/a)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user (No routes or components changed)

## Requested Feedback

Check that tests (unit) are passing.
Can verify other [content-build PR](https://github.com/department-of-veterans-affairs/content-build/pull/1873) functions [here](https://web-ihqrv6rby5lxlkn8zqdxf9u7lk1aplcp.demo.cms.va.gov/data/cms/vamc-system.json)

### CODE REVIEW

File: src/platform/site-wide/drupal-static-data/source-files/vamc-system/selectors/index.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// breaks unit tests if we use the workspace import
+// eslint-disable-next-line @department-of-veterans-affairs/use-workspace-imports
```
[the remark this answers is not in the document]
Comment (eselkin) [reply src/platform/site-wide/drupal-static-data/source-files/vamc-system/selectors/index.js:2]: Oddly if you fix this, tests break (I guess they can't find the relative imports to the @department-...)

File: src/platform/site-wide/drupal-static-data/source-files/vamc-system/tests/preProcess.unit.spec.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+/* eslint-disable camelcase */
```
[the remark this answers is not in the document]
Comment (eselkin) [reply src/platform/site-wide/drupal-static-data/source-files/vamc-system/tests/preProcess.unit.spec.js:1]: Data has snake case "vha_" prefixes

File: src/platform/site-wide/drupal-static-data/source-files/vamc-system/tests/utils.unit.spec.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
+/* eslint-disable camelcase */
```
[the remark this answers is not in the document]
Comment (eselkin) [reply src/platform/site-wide/drupal-static-data/source-files/vamc-system/tests/utils.unit.spec.js:1]: Data has snakecase "vha_" prefix

**Review by maxx1128 — APPROVED** [review]
This checks out for me, and I also learned the term "tuples." 😃

**Review by rsmithadhoc — APPROVED** [review]

**Pull request merged**